attestations.py

```diff
@@ -91,9 +91,12 @@ def main() -> None:
 
     # Make sure everything that looks like a dist actually is one.
     # We do this up-front to prevent partial signing.
-    for dist_path in dist_paths:
-        if not dist_path.is_file():
-            die(f'Path looks like a distribution but is not a file: {dist_path}')
+    if (invalid_dists := [_path for _path in dist_paths if _path.is_file()]):
```

Comment:
This is where the bug snuck in that needed fixing by https://github.com/pypa/gh-action-pypi-publish/commit/0ab0b79.

Reply:
🤦 thanks for fixing that.